README.md

```diff
@@ -13,11 +13,13 @@ A student representatives website. This project is live! You can see a working i
 To be able to contribute to 1327, you need to get the source code with all dependencies. Please note, that submodules for *bootstrap*, *font-awesome* and *bootstrap-markdown* are in use:
```

Comment:
this is not correct anymore

Reply:
How can it be corrected? Just by removing these two sentences?

Reply:
yes. bootstrap/fa are collected via yarn, so following those steps is enough to set everything up